velero-plugins/deploymentconfig/restore.go

```diff
@@ -58,23 +60,37 @@ func (p *RestorePlugin) Execute(input *velero.RestoreItemActionExecuteInput) (*v
 		}
 	}
 
-	// Set replicas to 0 if defaultVolumesToRestic is true
-
-	// get backup associated with the restore
-	backupName := input.Restore.Spec.BackupName
-	backup, err := common.GetBackup(input.Restore.GetUID(), backupName, input.Restore.Namespace)
-	if err != nil {
-		p.Log.Infof("[deploymentconfig-restore] could not fetch backup associated with the restore, got error: %s", err.Error())
-	}
-	var defaultVolumesToFsBackup *bool = nil
-	if err == nil {
-		// check for default fsbackup/restic flag
-		if boolptr.IsSetToTrue(backup.Spec.DefaultVolumesToRestic) || boolptr.IsSetToTrue(backup.Spec.DefaultVolumesToFsBackup) {
-			defaultVolumesToFsBackup = pointer.Bool(true)
+	// Set replicas to 0 if restoring pods
```

Comment:
Might be good to expand here why replicas 0 is required.

Reply:
Sure. The replicas=0 bit isn't new, but some explanation wouldn't be a bad thing.